pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1113,6 +1099,21 @@ func (og *operationGenerator) GenerateUnmapDeviceFunc(
 			}
 		}
 
+		// Execute tear down device
```

Comment:
Can you add a comment explaining that we need to call TearDownDevice afterwards because plugins that do local attach/detach will fail due to the refcnt on the loopback device.

Reply:
I will add a comment.